caas/broker.go

```diff
@@ -81,16 +81,24 @@ const (
 // ServiceType defines a service type.
 type ServiceType string
 
+// IsOmit indicates if a service is required.
+func (st ServiceType) IsOmit() bool {
+	return st == ServiceOmit
+}
+
 const (
 	ServiceCluster      ServiceType = "cluster"
 	ServiceLoadBalancer ServiceType = "loadbalancer"
 	ServiceExternal     ServiceType = "external"
```

Comment:
did we include "external" in the charm.v6 code?

Reply:
yes, we did. https://github.com/juju/charm/blob/v6/meta.go#L166